src/mpi/coll/bcast/bcast_intra_tree.c

```diff
@@ -0,0 +1,167 @@
+ /*
```

Comment:
Please remove the white space before the comment.

Reply:
Sorry I was occupied with other things. Will fix this tonight.